packages/psd/src/utils/bytes.ts

```diff
@@ -266,6 +266,16 @@ export class Cursor {
     return result;
   }
 
+  /**
+   * Reads unsigned 4-byte fixed-point number.
+   * 32 bits in 16.16 setup
+   * https://github.com/meltingice/psd.js/blob/333dd1467452a3353018c2856e3e4fb0e07d0025/lib/psd/resources/resolution_info.coffee#L10
+   */
+  readFixedPoint32bit(): number {
+    const int = this.read("u32");
+    return int / (1 << 16);
+  }
+
```

Comment:
The number of ways Adobe came up with to store numbers never ceases to surprise me. Good work👏